app/src/main/java/com/architect/coders/mu8/MainActivity.kt

```diff
@@ -1,5 +1,4 @@
 package com.architect.coders.mu8
-
```

Comment:
keep this blank line  (line2)

Reply:
In the commit [13f7e](https://github.com/Architect-Coders/MU8/commit/13f7e12201dc7d22673362ddbf8f735c6ada8a1burl) it is already corrected.